docs/parameters.rst

```diff
@@ -1,277 +1,179 @@
-- ``Radiative Forcing|Aerosols|BC`` (W/m\*\*2)
-- ``Radiative Forcing|Land-use Change`` (W/m\*\*2)
-- ``Radiative Forcing|Black Carbon on Snow`` (W/m\*\*2)
-- ``Radiative Forcing|Volcanic`` (W/m\*\*2)
-- ``Radiative Forcing|Solar`` (W/m\*\*2)
-- ``Radiative Forcing|External`` (W/m\*\*2)
-- ``Radiative Forcing|CO2`` (W/m\*\*2)
-- ``Radiative Forcing|CH4`` (W/m\*\*2)
-- ``Radiative Forcing|N2O`` (W/m\*\*2)
-- ``Radiative Forcing|NF3`` (W/m\*\*2)
-- ``Radiative Forcing|CF4`` (W/m\*\*2)
-- ``Radiative Forcing|C2F6`` (W/m\*\*2)
-- ``Radiative Forcing|C3F8`` (W/m\*\*2)
-- ``Radiative Forcing|cC4F8`` (W/m\*\*2)
-- ``Radiative Forcing|C4F10`` (W/m\*\*2)
-- ``Radiative Forcing|C5F12`` (W/m\*\*2)
-- ``Radiative Forcing|C6F14`` (W/m\*\*2)
-- ``Radiative Forcing|C7F16`` (W/m\*\*2)
-- ``Radiative Forcing|C8F18`` (W/m\*\*2)
-- ``Radiative Forcing|CCl4`` (W/m\*\*2)
-- ``Radiative Forcing|CHCl3`` (W/m\*\*2)
-- ``Radiative Forcing|CH2Cl2`` (W/m\*\*2)
-- ``Radiative Forcing|CH3CCl3`` (W/m\*\*2)
-- ``Radiative Forcing|CH3Cl`` (W/m\*\*2)
-- ``Radiative Forcing|CH3Br`` (W/m\*\*2)
-- ``Radiative Forcing|HFC23`` (W/m\*\*2)
-- ``Radiative Forcing|HFC32`` (W/m\*\*2)
-- ``Radiative Forcing|HFC4310`` (W/m\*\*2)
-- ``Radiative Forcing|HFC125`` (W/m\*\*2)
-- ``Radiative Forcing|HFC134a`` (W/m\*\*2)
-- ``Radiative Forcing|HFC143a`` (W/m\*\*2)
-- ``Radiative Forcing|HFC152a`` (W/m\*\*2)
-- ``Radiative Forcing|HFC227ea`` (W/m\*\*2)
-- ``Radiative Forcing|HFC236fa`` (W/m\*\*2)
-- ``Radiative Forcing|HFC245fa`` (W/m\*\*2)
-- ``Radiative Forcing|HFC365mfc`` (W/m\*\*2)
-- ``Radiative Forcing|CFC11`` (W/m\*\*2)
-- ``Radiative Forcing|CFC12`` (W/m\*\*2)
-- ``Radiative Forcing|CFC113`` (W/m\*\*2)
-- ``Radiative Forcing|CFC114`` (W/m\*\*2)
-- ``Radiative Forcing|CFC115`` (W/m\*\*2)
-- ``Radiative Forcing|HCFC22`` (W/m\*\*2)
-- ``Radiative Forcing|HCFC141b`` (W/m\*\*2)
-- ``Radiative Forcing|HCFC142b`` (W/m\*\*2)
-- ``Radiative Forcing|SF6`` (W/m\*\*2)
-- ``Radiative Forcing|SO2F2`` (W/m\*\*2)
-- ``Radiative Forcing|Halon1202`` (W/m\*\*2)
-- ``Radiative Forcing|Halon1211`` (W/m\*\*2)
-- ``Radiative Forcing|Halon1301`` (W/m\*\*2)
-- ``Radiative Forcing|Halon2402`` (W/m\*\*2)
+Standard parameters
+*******************
+
+Below we provide a list of the OpenSCM standard parameters adapters must adhere to as far as a specific variable concerns them.
+Alongside we give the type of unit that the parameter should be given in and how it should be expected by adapters.
+Conversion between particular units is done automatically if possible.
+
+In the following, ``<GAS>`` can be one of the standard :ref:`gases`.
+
+.. csv-table:: Standard parameters
+    :header: "Parameter name 0", "Parameter name 1", "Parameter name 2", "Unit type", "Note"
+
+    ``Emissions``, ``<GAS>``,, "mass <GAS> / time"
+    ``Atmospheric Concentrations``, ``<GAS>``,, "concentration (e.g. parts per X)", "Aggregation possible, but does not always make sense"
+    ``Pool``, ``<GAS>``,, "mass <GAS>"
```

Comment:
I wonder if we can do better than 'Pool', maybe not...

Reply:
Hmm, it's the standard expression, no? "Gas pool"?

Reply:
I think so, it just seems unsatisfactory that it is so...